README.md

```diff
@@ -130,6 +130,7 @@ The stack has a `SecretsBucket` parameter which will allow your build agents to
 
 The secrets bucket can contain the following files:
 
+* `/env` - An optional bash script to use as a global [agent environment hook](https://buildkite.com/docs/agent/hooks)
```

Comment:
> Happy to merge this though, it is at least consistently incorrect.

yeah, I saw all those evals and I was thinking on fixing them in another PR... 

FWIW I think that replacing `eval $(s3_download file)` for `source <(s3_download file)` should work the same way as before (assuming no eval-dependent hacks)